fastapi_server/app/main.py

```diff
@@ -72,4 +86,207 @@ async def get_orders():
         return JSONResponse(content=json.loads(json_util.dumps(all_orders)))
     finally:
         client.close()
-        print("Connection to MongoDB closed.")
\ No newline at end of file
+        print("Connection to MongoDB closed.")
+
+def get_db_connection():
```

Comment:
Use the already created - def get_cockroach_db_connection(db_name): in the start